portfolio/src/main/webapp/script.js

```diff
@@ -16,8 +16,10 @@
  * Adds a random greeting to the page.
  */
 function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+  const greetings = 
+    ['I love to travel!', 'I enjoy learning about skincare and multimedia communication.',
+     'My favorite musical artist is Rihanna.', 'My resolution was to give up boba this year, and I have yet to cave!'];
+     //['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
```

Comment:
Feel free to delete this comment now that you're not using it. It's good practice to remove so-called "dead code" that isn't used to help keep the code base neat. Otherwise, in a couple of months or years, you might look at this and have no idea why it's here :)